File: MTWDM iOS Xamarin/AppSQLite/AppSQLite/customTextField.cs
using Foundation;
using System;
using UIKit;
using CoreGraphics;

namespace AppSQLite
{
    public partial class customTextField : UITextField
    {
        public customTextField (IntPtr handle) : base (handle)
        {
            configurarPropiedades();
        }

        private void configurarPropiedades()
        {
            this.BorderStyle = UITextBorderStyle.None;
            this.Layer.BorderColor = UIColor.Blue.CGColor;
            this.Layer.BorderWidth = 0.5f;
            this.Layer.CornerRadius = 20;

            var f = this.Frame;
            f.Height = 40f;
            this.Frame = f;

            this.Font = UIFont.FromName("Arial", 18f);

        }

        public override CGRect TextRect(CGRect forBounds)
        {
            var padding = new UIEdgeInsets(0, 10, 0, 0);

            return base.TextRect(padding.InsetRect(forBounds));
        }

        public override CGRect EditingRect(CGRect forBounds)
        {
            return TextRect(forBounds);
        }
    }
}

File: MTWDM iOS Swift /appSQLITE/appSQLITE/customTextField.swift
//
//  customTextField.swift
//  appSQLITE
//
//  Created by Felipe Hernandez on 12/7/18.
//  Copyright © 2018 Felipe Hernandez. All rights reserved.
//

import UIKit

class customTextField: UITextField {
    
    /*
     Prepara el receptor para el servicio después de que se haya cargado desde un archivo de Interface Builder o un archivo nib.
     
     */
    override func awakeFromNib() {
        super.awakeFromNib()
        
        configurarPropiedades()
        
    }

    func configurarPropiedades() {
        
        self.borderStyle = .none
        self.layer.borderColor = UIColor.blue.cgColor
        self.layer.borderWidth = 0.5
        self.layer.cornerRadius =  15
       
        self.frame.size = CGSize(width: self.frame.width, height: 40)
        
    }
    
    @IBInspectable var padding: CGFloat = 0
    
    override func textRect(forBounds bounds: CGRect) -> CGRect {
        return CGRect(x: bounds.origin.x + padding, y: bounds.origin.y, width: bounds.width - padding * 2, height: bounds.height)
    }
    
    override func editingRect(forBounds bounds: CGRect) -> CGRect {
        return CGRect(x: bounds.origin.x + padding, y: bounds.origin.y, width: bounds.width - padding * 2, height: bounds.height)
    }
}
